{"name": "test_search_with_valid_input", "status": "broken", "statusDetails": {"message": "AttributeError: 'SubRequest' object has no attribute 'mode'. Did you mean: 'node'?", "trace": "self = <allure_commons._allure.fixture object at 0x0000022CD43A16D0>, args = (), kwargs = {}\n\n    def __call__(self, *args, **kwargs):\n        self.parameters = func_parameters(self._fixture_function, *args, **kwargs)\n    \n        with self:\n>           return self._fixture_function(*args, **kwargs)\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\n.venv\\Lib\\site-packages\\allure_commons\\_allure.py:231: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nrequest = <SubRequest 'log_on_failure' for <Function test_search_with_valid_input>>\n\n    @pytest.fixture()\n    def log_on_failure(request):\n        yield\n>       item=request.mode\n             ^^^^^^^^^^^^\nE       AttributeError: 'SubRequest' object has no attribute 'mode'. Did you mean: 'node'?\n\ntests\\conftest.py:11: AttributeError"}, "start": 1758948624472, "stop": 1758948625019, "uuid": "f90029cf-0509-4411-93ac-7867b2d1483f", "historyId": "2699a69d374bdf758d5a5adc211b265d", "testCaseId": "2699a69d374bdf758d5a5adc211b265d", "fullName": "tests.test_Search.TestSearch#test_search_with_valid_input", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_Search"}, {"name": "subSuite", "value": "TestSearch"}, {"name": "host", "value": "DESKTOP-PQJ2IUR"}, {"name": "thread", "value": "12224-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_Search"}], "titlePath": ["tests", "test_Search.py", "TestSearch"]}